OpenEdXMobile/src/main/java/org/edx/mobile/util/observer/EventObserver.kt

```diff
@@ -10,8 +10,9 @@ import androidx.lifecycle.Observer
  * Inspiration: https://gist.github.com/JoseAlcerreca/e0bba240d9b3cffa258777f12e5c0ae9
  */
 class EventObserver<T>(private val consumer: (T) -> Unit) : Observer<Event<T>> {
-    override fun onChanged(event: Event<T>?) {
-        consumer(event?.getContentIfNotConsumed() ?: return)
+
+    override fun onChanged(value: Event<T>) {
```

Comment:
Any particular reason for this change?

Reply:
Recommended by Android Studio, after updating `GRADLE_PLUGIN_VERSION`.